### DISCUSSION TITLE
request for dark mode

### DISCUSSION THREAD

**Discussion by fahemboughanem** [sumatrapdfreader/sumatrapdf#2960] [Documentation]
Sumatara pdf it is a good but shortage a dark mode

**Comment by GitHubRulesOK** [comment]
corrrect there is NO mode control it is a canvas with system surround but the UI rendering code was written by MS VS for XP+ and UI mode settings are controlled by windows such that 3rd party WindowsBlinds can alter to mood
![image](https://user-images.githubusercontent.com/12754913/190857007-1c050832-48d9-4592-b773-1063288dfd1f.png)

**Comment by jolyont** [comment]
I'd like this feature too.   Its one of the few non-dark apps that I use now.

It looks like we can sort of do this by messing around in advanced settings and editting the settings file, but I can't seem to change the surround.  I changed: MainWindowBackground, TextColor and BackgroundColor.

**Reply by GitHubRulesOK** [reply]
@jolyont the demo screen above shows its a 3rd Party framework
see the comment those "surrounds" are "system colours" (worked fine in XP), but MS totally altered the way they are handled in the switch to WimpDos Metro8 You can have any colour as long aas its handled by the system from white to 50 shades of ...
